service/src/main/resources/templates/index.html

```diff
@@ -122,12 +138,13 @@
       ],
       plugins: [
         SwaggerUIBundle.plugins.DownloadUrl,
-        pinLoginPlugin,
-        clearValidator
+        clearValidator,
+        LoadExternalValuePlugin
       ],
       layout: 'StandaloneLayout',
-      displayOperationId: true,
-      docExpansion: 'none',
+      displayRequestDuration: true,
+      tryItOutEnabled: true,
+      defaultModelsExpandDepth: 0,
```

Comment:
These are a a few additional changes I added while working on this but are not required for this change.

- I removed `docExpansion: 'none'` because I want to have the top level tag nodes expanded by default
- `displayRequestDuration: true` shows the request duration for "try it out" requests which seems like a nice feature to have
- `tryItOutEnabled: true` changes the swagger behavior back to the old behavior, where "try it out" was enabled by default. So with this change you don't have to do an extra click before using Try it out.
- `defaultModelsExpandDepth: 0` keeps the schema models collapsed even when the tag groups are expanded.

I also removed `displayOperationId: true`. This hides the operation ID of a swagger command from the web page. So for example, the `/status` endpoint would no longer show `getStatus` next to it. The ID is really an internal thing so it seems cleaner to me to hide it, but I can add that back if others thing would be better to leave it in.

For a full list of swagger configuration options: https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/